### ISSUE TITLE
Nouvelle source pour films/séries

### ISSUE THREAD

**Issue by TheSamurai4861** [Kodi-vStream/venom-xbmc-addons#4250]
Bonjour,

J'ai récemment découvert un site qui permet d'obtenir un lien de streaming pour n'importe quel film/série ayant un id TMDB ou IMDB. 

Voici l'url : https://frembed.pro/

Tout est expliqué sur le site mais voici une brève explication : 

Pour un film tel que Titanic, l'id TMDB est 597.
Pour récuperer le lien de streaming, il suffit de faire la requête suivante : https://frembed.pro/api/film.php?id=597

Pour une série tel que Les Simpsons, avec l'id 456 S01E01, il faut faire la requête suivante : https://frembed.pro/api/serie.php?id=456&sa=1&epi=1

Cette solution est très pratique car elle ne se base que sur l'id TMDB du média.

Ils proposent d'ailleurs un CSV avec l'ensemble des films et séries dispos sur leur site.

**Comment by TheSamurai4861** [comment]
Oui, il faudrait essayer un film dans vStream pour voir s'il arrive à déchiffrer leur lecteur. 

Ou alors, tenter d'analyser leur lecteur pour récupérer les liens Voe, Dood, ...

**Comment by sizanic** [comment]
Bonjour,

En effet, on nous avait déjà parlé de ce site, mais on ( je ) n'avait pas prit le temps de s'en occuper.

Je vais le remettre en haut des priorités, vu le potentiel.

**Comment by sizanic** [comment]
Je suis en train de préparer une nouvelle version de vStream. J'espère pouvoir mettre ce site dedans 🙏🏻

**Comment by sizanic** [comment]
L'accès aux liens est vraiment bien protégé.
Je me suis cassé les dents, impossible de récupérer le lien de streaming 😢

**Comment by TheSamurai4861** [comment]
@sizanic pas de problèmes. C'est gentil d'avoir essayer.

**Comment by sviet2k** [comment]
@sizanic 
Comment ca va?

Je pense pour recuperer les liens, il faut juste rouler cette requete:
   hxxps://frembed.lol/api/films?id=597&idType=tmdb
et la reponse contient tous les liens

**Comment by sizanic** [comment]
C'est avec les liens de la réponse que je suis ensuite bloqué.
Il ne me semble pas qu'ils soient directement accessibles.

**Comment by sviet2k** [comment]
URL resolver addon peut trouver les "direct" liens pour ces hosts trouves.
Je me souviens plus: vstream utilise URL Resolver ? Si c'est non donc oublie ce site :(

**Comment by sizanic** [comment]
vStream a longtemps été compatible avec URLResolver, mais ça fait longtemps que plus aucune source ne l'utilise.
Il faudrait tester.

**Comment by TheSamurai4861** [comment]
Je confirme que URL resolver marche encore pour la plupart des sources que vStream utilise.

**Comment by TheSamurai4861** [comment]
J'ai réussi à résoudre Dood, Voe, Sibnet, Uqload, ...

**Comment by GravuTrad** [comment]
Des news sur cette piste?

**Comment by sizanic** [comment]
C'est en cours 👍🏻

**Comment by GravuTrad** [comment]
Autre idée, beaucoup de ces sites mettent leur adresse à jour sur leur discord. (cela me sert quand leur redirection ne peut pas marcher)
Pas moyen de le récupérer automatiquement là bas?

**Comment by GravuTrad** [comment]
Merci pour l'ajout de cinestream.info sur la v4.
J'ai du décocher cpasmal et wiflix qui bloquaient.

Sinon d'autres sites sympa trouvés:
https://purstream.mx
avec son index site ici: https://purstream.wiki
et pour les séries uniquement; https://papadustream.cash
et : https://gadrov.com

**Comment by TmpName** [comment]
>J'ai du décocher cpasmal et wiflix qui bloquaient.

Ca devrait plus bloquer avec la nouvelle version de Cloudscraper ?
Gadrov deja sur vstream, Kepliz

**Comment by GravuTrad** [comment]
non ok vu, cpasmal et wiflix bloquent sur la recherche sans vpn et avec vpn la recherche passe. pourtant mon service dns n'est pas un des habituels et n'est pas bloquant. D'habitude pas besoin du vpn. étrange.
